spec/controllers/container_group_controller_spec.rb

```diff
@@ -36,4 +36,25 @@
     expect(response.status).to eq(200)
     expect(response.body).to_not be_empty
   end
+
+  describe "#show" do
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+      @container_group = FactoryGirl.create(:container_group)
+      @user = FactoryGirl.create(:user)
+      login_as @user
+    end
+
+    subject { get :show, :id => @container_group.id }
+
+    context "respond_with" do
+      it { is_expected.to have_http_status 200 }
+    end
+
+    context "render listnav partial" do
+      render_views
```

Comment:
I looked to your PR and I still see there 2 GET requests. Difference is only that you had not separated contexts (if i get you right, you mean this change - martinpovolny/manageiq@037a914)